src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const REVERSE_ROBOT = {};
+  const VALUES = Object.values(robot);
+  const KEYS = Object.keys(robot);
+  const REMOVE_DUBLICATE_VALUES = new Set(VALUES);
+
+  if ([...REMOVE_DUBLICATE_VALUES].length !== VALUES.length) {
+    return null;
+  }
+
+  for (let i = 0; i < KEYS.length; i++) {
+    REVERSE_ROBOT[VALUES[i]] = KEYS[i];
+  }
```

Comment:
The loop that creates the `REVERSE_ROBOT` object is correct, but it could be done more efficiently. Instead of looping over the `KEYS` array and accessing the `VALUES` array by index, you could loop over the `robot` object's entries using `Object.entries()`. This would give you both the key and the value in each iteration, making the code cleaner and more efficient.